src/testimonials/testimonials.resolver.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  InternalServerErrorException,
+  Logger,
+  UseGuards,
+} from '@nestjs/common';
+import { AuthGuard } from '../auth/auth.guard';
+import { JwtProcessorType } from '../auth/auth.service';
+import { JwtType } from '../auth/jwt/jwt.type.decorator';
+import { Query, Mutation, Resolver, Args, Int } from '@nestjs/graphql';
+import { Testimonial } from './api/testimonial.model';
+import { TestimonialDto } from './api/TestimonialDto';
+import { TestimonialsService } from './testimonials.service';
+import { CreateTestimonialRequest } from './api/CreateTestimonialRequestREST';
+import {
+  API_DESC_CREATE_TESTIMONIAL,
+  API_DESC_GET_TESTIMONIALS,
+  API_DESC_GET_TESTIMONIALS_ON_SQL_QUERY,
+} from './testimonials.controller.api.desc';
+
+@Resolver((of) => Testimonial)
```

Comment:
Remove unused parameters  

```suggestion
@Resolver(() => Testimonial)
```